cmake/GNU.cmake

```diff
@@ -1,28 +1,27 @@
-
 set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fcray-pointer -ffree-line-length-none -fno-range-check -fbacktrace")
```

Comment:
can we add `-g -fbacktrace` here?

Reply:
Should we use `-g` or `-ggdb`?

Reply:
either will work. `-g` is portable. `-ggdb` will produce information for use by GDB.  Not sure about the portability of `-ggdb`.  I would prefer the latter.